Sources/SmileID/Classes/Helpers/AnyDecodable.swift

```diff
@@ -0,0 +1,192 @@
+// swiftlint:disable all
+#if canImport(Foundation)
+import Foundation
```

Comment:
Any reason why this should be possible to import?

Reply:
I will remove the conditional check.